hooks/pre-push

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+echo "Running linter tests"
+lines=$(sh scripts/run_linter.sh | wc -l)
+if [ $lines -eq 1 ]; then
```

Comment:
I think looking at the exit codes would be a more direct signal. See how travis does it.